vendor/parser-extended/parser.mly

```diff
@@ -2201,8 +2201,11 @@ expr:
 ;
 
 simple_expr:
-  | LPAREN seq_expr RPAREN
-      { reloc_exp ~loc:$sloc $2 }
+  | LPAREN e = seq_expr RPAREN
+      { match e.pexp_desc with
+        | Pexp_pack _ ->
+            mkexp ~loc:$sloc (Pexp_parens e)
+        | _ -> reloc_exp ~loc:$sloc e }
```

Comment:
I made this dirty fix (introducing a new node Pexp_parens only in this case) to preserve the location of comments here. As this is the direction we are taking anyway.
@Julow  @EmileTrotignon what do you think?

If you are okay with introducing this new node in the ast now, we can do it in another PR to not pollute this one.

Reply:
I am fine with this